lib/cache-driver.js

```diff
@@ -137,10 +137,6 @@ function CacheDriver(callContext, authority, resource, clientId, cache, refreshF
  * @param  {QueryCallback} callback
  */
 CacheDriver.prototype._find = function(query, callback) {
-  var authorityQuery = {};
```

Comment:
We firstly find all potential entries, and then find tentant and resource specific entry. 
check here https://github.com/AzureAD/azure-activedirectory-library-for-nodejs/pull/83/files#diff-8219131514daab4a7f8d1b3f58959918R204